lib/scripts/pdf_a4_portrait.js

```diff
@@ -153,11 +154,19 @@ function createSection (section, content, options) {
     height: options.height,
     contents: phantom.callback(function (pageNum, numPages) {
       var html = o[pageNum] || c[pageNum]
+
+      var pageNumFinal = pageNum, numPagesFinal = numPages;
+      if(opts.paginationStartPage && opts.paginationStartPage > 1) {
+        var pageOffset = opts.paginationStartPage - 1;
+        pageNumFinal = (pageNumFinal - pageOffset > 0 ? pageNumFinal - pageOffset : '');
```

Comment:
Will do. The empty string is to actualy print out a blank for cover pages and such